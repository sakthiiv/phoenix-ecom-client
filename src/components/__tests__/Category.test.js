import React from "react";
import { shallow } from "enzyme";
import Categoty from "../Category";


const category = {
	"id": "5bc9a87d76e3833f73511a17",
	"description": "Hi Test",
	"name": "Electronics Test",
	"subCategory": [{
		"id": "5bd0526764151f2c94f622aa",
		"description": "Hi Sub",
		"name": "nameSub",
		"subCategory": null,
		"valid": null
	}],
	"valid": true
}
describe("Categoty", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<Categoty category={category}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
