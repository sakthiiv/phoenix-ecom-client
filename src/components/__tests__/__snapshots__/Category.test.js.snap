// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categoty renders correctly 1`] = `
<Link
  replace={false}
  to={
    Object {
      "pathname": "/userproductList",
      "state": Object {
        "category": Object {
          "description": "Hi Test",
          "id": "5bc9a87d76e3833f73511a17",
          "name": "Electronics Test",
          "subCategory": Array [
            Object {
              "description": "Hi Sub",
              "id": "5bd0526764151f2c94f622aa",
              "name": "nameSub",
              "subCategory": null,
              "valid": null,
            },
          ],
          "valid": true,
        },
      },
    }
  }
>
  <div
    className="list-container cat-list"
    key="5bc9a87d76e3833f73511a17"
  >
    <Row
      tag="div"
    >
      <Col
        sm="12"
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Card
          tag="div"
        >
          <CardBody
            tag="div"
          >
            <div
              className="cart-title-container"
            >
              <div
                className="cart-title-left"
              >
                <h3>
                  Electronics Test
                </h3>
              </div>
            </div>
          </CardBody>
          <CardBody
            className="card-body"
            tag="div"
          >
            <CardText
              tag="p"
            >
              Hi Test
            </CardText>
          </CardBody>
        </Card>
      </Col>
    </Row>
  </div>
</Link>
`;
